src/runtime.cpp

```diff
@@ -198,11 +198,11 @@ namespace hpx
         };
```

Comment:
This array of state names needs adaptation as well (see the function `get_runtime_state_name()` below).